recipes/maeparser/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "maeparser" %}
+{% set version = "1.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/schrodinger/maeparser/archive/v{{ version }}.tar.gz
+  sha256: a8d80f67d1b9be6e23b9651cb747f4a3200132e7d878a285119c86bf44568e36
+  patches:
+    - patches/001-patch-cmake-config-path.patch
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ stdlib('c') }}
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [not win]
+  host:
+    - libboost-devel
+    - zlib
+
+test:
+  requires:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [not win]
+    - libboost-devel
+    - zlib
+  files:
+    - test.cpp
+    - CMakeLists.txt
+    - test_maeparser.sh  # [unix]
+    - test_maeparser.bat  # [win]
```

then you don't need the test scripts anymore either
```suggestion
    - cmake_test/
```